cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/StreamUtils.java

```diff
@@ -423,6 +424,33 @@ public Long getResult() {
     return processor.getResult();
   }
 
+  /**
+   * Constructs a table name for stream consumer state stores within a given namespace.
+   * @param namespace the namespace for which the table is for.
+   * @return constructed table name
+   */
+  public static String constructStateStoreTableName(Id.Namespace namespace) {
```

Comment:
In this case, `get`, not `construct`. BTW, as a reference use `create` instead of `construct`.